### ISSUE TITLE
Stop-Limit Order

### ISSUE THREAD

**Issue by ciottied** [JKorf/Bitfinex.Net#16]
Hi there.
Two questions:
1) There is a way to insert Stop-Limit Order ? Order.enum does'nt seem to have a stopLimit value.
2) What's the difference between ExhangeLimit and Limit, ExhangeStop-Stop ?

Thank you
BTW : You did really a great job with the library.

**Comment by ciottied** [comment]
BTW, running in debug mode in VS 2017, immediate window prints:
--------------------------------------------------------------------------
2018/05/11 10:40:20:781 | Info | Socket connection established
2018/05/11 10:40:20:864 | Info | Websocket version: 2, platform status: operational
2018/05/11 10:40:20:993 | Info | Authentication successful
Cannot map enum. Type: Bitfinex.Net.Objects.OrderType, Value: EXCHANGE STOP LIMIT
Cannot map enum. Type: Bitfinex.Net.Objects.OrderType, Value: EXCHANGE STOP LIMIT
--------------------------------------------------------------------------
There is a issue withe the library ?

**Comment by Espleth** [comment]
> What's the difference between ExhangeLimit and Limit, ExhangeStop-Stop ?
Exchange* orders are for exchange trading, and other are for margin trading. But I agree, it needs clarification.

**Comment by JKorf** [comment]
Hi, Thanks for the kind words.

As @Espleth mentioned, the difference between ExhangeLimit and Limit is to differentiate between exchange trading and margin trading. This is just how Bitfinex has named these values, but I agree it should be a bit more clear, I'll maybe change it later. 

Regarding the warning you got; the StopLimit ordertype wasn't mapped to an enum, I fixed that in the last release. 

ExchangeStopLimit orders aren't supported by the WebApi, but can be placed with the websocket client. Use the priceAuxiliaryLimit for this.

**Comment by ciottied** [comment]
@Espleth , @JKorf  Thank you guys.
Suggestion for Order Enum: ExchangeLimit, MarginLimit, ExchangeStop, MarginStop.

Stop Limit Order: I tried with V2API websocket order with stop price in price field and limit in AuxiliaryLimit. Got en error, something "so response for server".
But I tried just once in debug mode, and possibily it was  a timeout error or I screwed something. Will try again and report.
